tests/test_load_dump.py

```diff
@@ -599,3 +600,25 @@ def test_float_subclass(self):
             rate=decimal.Decimal("0.000012345"),
         )
         self.check_dump_load_roundtrip(b.resolve())
+
+    def test_json_infinities_get_stringified(self):
+        re_inf = re.compile(r'[^"]\bInfinity\b[^"]')
```

Comment:
What are the first two blocks testing? Doesn't seem like it's anything to do with demes.